Cargo.toml

```diff
@@ -48,4 +48,7 @@ tabular = "0.1.4"
 base64 = "0.13.0"
 rcgen = { version  = "0.8.11", features = ["pem", "x509-parser"] }
 x509-parser = "0.9.2"
-json_value_merge = "0.1.2"
\ No newline at end of file
+json_value_merge = "0.1.2"
+
+rsa = "0.5.0"
```

Comment:
would it work with "0.3.0"  (0.5.0 is not shipped in fedora)

Reply:
No, but "0.4.0" works. Can we use that? I feel bad that I didn't notice it before and waited for 5.0 :)

Reply:
No don't worry, I'll submit an update for 0.5.0 :)

Reply:
Okay, thanks!